File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        data = {}
        for i in nums:
            data[i]=0
        for i in nums:
            data[i]+=1
            
        for i in data:
            if data[i]>1:
                return True
        return False

        

File: Data Structures & Algorithms/special-array-i/submission-1.ts
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    isArraySpecial(nums: number[]): boolean {
        for(let i = 0;i<nums.length-1;i++){
            // console.log(nums[i],nums[i+1],(nums[i]+nums[i+1])%2==0)
            if((nums[i]+nums[i+1])%2===0){
                return false;
            }
        }
        return true;
    }
}
